modules/openapi-generator/src/main/java/org/openapitools/codegen/InlineModelResolver.java

```diff
@@ -627,6 +653,30 @@ private Schema modelFromProperty(OpenAPI openAPI, Schema object, String path) {
         model.setXml(xml);
         model.setRequired(object.getRequired());
         model.setNullable(object.getNullable());
+        model.setDiscriminator(object.getDiscriminator());
```

Comment:
So I have some questions about how our code works here. Can you help answer them?
1. This copies the property from the inline schema into the extracted schema that we put into openapi component schemas, right?
So it sets the discriminator on this schema:
```
      - type: object
        discriminator: party_type
        required:
          - party_type
        properties:
          party_type:
            readOnly: true
            $ref: '#/definitions/PartyType'
          tax_id_number:
            type: string
```
Which is extracted and used to create the schema with the key `Party_allOf`, correct?
2. Is only one Party_allOf class made for each composed schema or is each inline schema extracted out into its own Party_allOf1, Party_allOf2 etc?

Reply:
`Party_allOf` will be created once, then any properties it references will be flattened. Later, we "lookup" the transient `Party_allOf` by serializing and deserializing the structure as the key of a map, and the value as a name. This is how we basically deduplicate those dynamic structures and associate them all with a generated name. See `structureMapper` in this PR which I've updated to have determinate property order. As it was, Jackson would probably serializer in the same way maybe 99% of the time, but without explicitly sorting it'll follow the same rules as JSON where map properties are technically unordered.